tests/models/qwen2_5/colqwen2_5/test_processing_colqwen2_5.py

```diff
@@ -0,0 +1,52 @@
+from typing import Generator, cast
+
+import pytest
+import torch
+from PIL import Image
+
+from colpali_engine.models import ColQwen2_5_Processor
+
+
+@pytest.fixture(scope="module")
+def model_name() -> str:
+    return "vidore/colqwen2.5-v0.2"
```

Comment:
See previous comment for reference.

```suggestion
    return "vidore/colqwen2.5-v1.0"
```